ComfortZoneApp/app/src/main/res/layout/fragment_input.xml

```diff
@@ -21,12 +21,12 @@
 
     <androidx.cardview.widget.CardView
         android:layout_width="match_parent"
-        android:layout_height="200dp"
+        android:layout_height="300dp"
```

Comment:
Even fo the commit message for this commit, you could have written "update the profile and input fragment layout"